weasyprint/__init__.py

```diff
@@ -181,14 +183,20 @@ def __init__(self, guess=None, filename=None, url=None, file_obj=None,
         self.etree_element = self.wrapper_element.etree_element
 
     def _ua_stylesheets(self, forms=False):
+        from .html import HTML5_UA_FORM_STYLESHEET, HTML5_UA_STYLESHEET
+
         if forms:
             return [HTML5_UA_STYLESHEET, HTML5_UA_FORM_STYLESHEET]
         return [HTML5_UA_STYLESHEET]
 
     def _ua_counter_style(self):
+        from .html import HTML5_UA_COUNTER_STYLE
+
         return [HTML5_UA_COUNTER_STYLE.copy()]
 
     def _ph_stylesheets(self):
+        from .html import HTML5_PH_STYLESHEET
```

Comment:
Noticed with failing tests that I also had to import them in functions as well. So the `__getattr__` machinery I added is only there for backwards compatibility in case e.g. users are importing `from weasyprint import HTML_UA_COUNTER_STYLE`